source/devices/AM62X/android/Release_Specific_Release_Notes.rst

```diff
@@ -61,13 +61,13 @@ SDK Components and Versions
 +------------------------------------+-------------------------------------------------------------------------------+
 | **Linux Kernel**                   | Linux 6.6.56                                                                  |
 +------------------------------------+-------------------------------------------------------------------------------+
-| **Linux Kernel Toolchain**         | clang-r487747c                                                                |
+| **Linux Kernel Toolchain**         | clang-r510928                                                                 |
 +------------------------------------+-------------------------------------------------------------------------------+
 | **U-boot**                         | 2024.04                                                                       |
 +------------------------------------+-------------------------------------------------------------------------------+
 | **GCC Toolchain**                  | GNU Toolchain for the A-profile Architecture 13.3 (rel1)                      |
 +------------------------------------+-------------------------------------------------------------------------------+
-| **Base Linux SDK**                 | PROCESSOR-SDK-LINUX-AM62X 10.01.08                                            |
+| **Base Linux SDK**                 | PROCESSOR-SDK-LINUX-AM62X 10.01.10                                            |
 +------------------------------------+-------------------------------------------------------------------------------+
 | **Hardware Supported**             | AM62X-SK / AM62X-LP / AM62B-P1 / Beagle Play                                  |
 +------------------------------------+-------------------------------------------------------------------------------+
```

Comment:
while at it, can you fixup this hw supported line item as well to call out [AM62P](https://www.ti.com/tool/SK-AM62P-LP)

Reply:
Not sure I understand. This file is only for the AM62X SDK documentation.

We mark AM62P as supported in `processor-sdk-doc/source/devices/AM62PX/android/Release_Specific_Release_Notes.rst`

Do I need to reword `AM62PX-SK` to something else? (Like `AM62P-SK`) ?

Reply:
my bad . looks you have it in the right place, i was looking at wrong file. approving .